tests/integration/cartography/intel/aws/test_route53_sync.py

```diff
@@ -39,6 +39,18 @@ def test_sync_route53(mock_get_zones, neo4j_session):
     create_test_account(neo4j_session, TEST_ACCOUNT_ID, TEST_UPDATE_TAG)
     _ensure_local_neo4j_has_test_ec2_records(neo4j_session)
 
+    # Create IP nodes that DNS records will point to
+    neo4j_session.run(
```

Comment:
This is a hack that I will undo after we properly figure out cartography.intel.dns